docs/cloud/satellites/gha-runners.md

```diff
@@ -1,142 +1,103 @@
-# Satellites as GHA runners (**experimental**)
+# Satellites as GitHub Actions runners (**experimental**)
```

Comment:
Note that this title doesn't actually show up in GitBook (it uses the title in the SUMMARY.md file).

I would maybe add a warning box at the top to say that this is experimental.

Reply:
done